packages/analytics/client/src/analytics_client/analytics_client.ts

```diff
@@ -360,3 +360,39 @@ export class AnalyticsClient implements IAnalyticsClient {
       });
   }
 }
+
+/** security paths that contain user data */
+const SECURITY_PATHS = ['hosts/name', 'users/name', 'network/ip'];
```

Comment:
Can we generalize the name of the variable so other solutions can add their paths to the list whenever they notice them?